src/main/java/io/stargate/sgv2/jsonapi/api/configuration/ObjectMapperConfiguration.java

```diff
@@ -25,6 +25,10 @@ ObjectMapper objectMapper(
       customizer.customize(mapper);
     }
 
+    // 14-Aug-2024, tatu: Our error handling assumes that by default we do fail on
+    //    unrecognized properties: ensure it's not disabled
+    mapper.enable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES);
```

Comment:
So: handling by Quarkus changed somewhere between 3.9 and 3.13 so that Quarkus started disabling this feature... and we need to restore it.